Hyperbolic_surface_triangulation_2/test/Hyperbolic_surface_triangulation_2/hs_test_circular_kernel.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2024
+// INRIA Nancy (France), and Université Gustave Eiffel Marne-la-Vallee (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org)
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Vincent Despré, Loïc Dubois, Monique Teillaud
+
+#include <iostream>
```

Comment:
Recently we found errors where CGAL header files had did miss to include what they depend on. 
In order to test that we should include in test files the new headers before including number types, std files etc.
So below lines 22-25 should be moved before line 13.

Reply:
done

Reply:
done